.vscode/settings.json

```diff
@@ -21,5 +21,4 @@
     "python.testing.cwd": "${workspaceFolder}/code",
     "python.testing.unittestEnabled": false,
     "python.testing.pytestEnabled": true,
-    "pylint.path" : [ "${interpreter}", "-m", "pylint" ]
```

Comment:
This prevents pylint from working in the DevContainer - we should find a solution for both. @liammoat - Assume you added this to get this working in a Mac